README.md

```diff
@@ -116,7 +116,8 @@ Clone or download the Identity Apps repository and run the following commands fr
 
 ```shell
 # From project root.
-mvn clean install
+pnpm install
```

Comment:
We may still use the maven command to build thentire repo as it's required for the JSP portals.

Reply:
Fixed with a428ddb0469faffee00b49fa1dd3a9a50df2fe36